lib/hermit-git

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+#
+# Summary: Run git operations on current profile repository
+# Usage: hermit git <git arguments>
+
+# This is a pass-through from hermit to git. It doesn't implement
+# any additional functionality, just changes to the profile directory
+# and runs git with the arguments specified.
+#
+# Any commands that implement additional functionality should be broken
+# out of this file and implemented as their own wrappers to what git
+# provides.
+#
```

Comment:
Similarly, unless you want the copyright to be part of the help doc, this should be a blank line.